compatibility/bazel_tools/daml_sdk.bzl

```diff
@@ -14,21 +16,25 @@ source "${RUNFILES_DIR:-/dev/null}/$f" 2>/dev/null || \
 """
 
 def _daml_sdk_impl(ctx):
+    # The DAML assistant will mark the installed SDK read-only.
+    # This breaks Bazel horribly on Windows to the point where
+    # even `bazel clean --expunge` fails because it cannot remove
+    # the installed SDK. Therefore, we do not use the assistant to
+    # install the SDK but instead simply extract the SDK to the right
+    # location and set the symlink ourselves.
```

Comment:
Hmm, isn't there a risk of these tests testing something else than what users what experience? E.g. if the DAML assistant performs some configuration at installation at some point. Would it work to remove the read-only flag after install? Maybe in a trap to avoid leaving the bazel tree in a bad state if installation fails.

Reply:
Yes there is a small risk here but after having spend way too much time trying to make this work otherwise, this seemed like the most reasonable option to me. Testing the SDK installation isn’t really a compatibility thing anyway and we do still have the actual SDK integration tests for that (at least for now).